integration/zone_aware_test.go

```diff
@@ -119,7 +125,7 @@ func TestZoneAwareReplication(t *testing.T) {
 	// Push 1 more series => all good
 	numSeries++
 	metricName = fmt.Sprintf("series_%d", numSeries)
-	series, expectedVector, _ = generateSeries(metricName, now)
+	series, expectedVector, _ = generateHistogramSeries(metricName, now)
```

Comment:
Does this need to be a histogram series for this test?

Reply:
Same as above, we're just trying to check both kinds where possible. For the zone aware test, we are alternating between floats and histograms manually (in other parts it's `generateFloatSeries`).